draft-ietf-trans-rfc6962-bis.md

```diff
@@ -681,6 +678,30 @@ value will not be a valid X.509v3 signature that could be used in conjunction
 with the TBSCertificate (from `SignedData.encapContentInfo.eContent`) to
 construct a valid certificate.
 
+### Binding Intent to Issue  {#binding_intent_to_issue}
+
+Under normal circumstances, there will be a short time period between
+precertificate submission and issuance of the corresponding certificate.
+Occasional delays are to be expected though (e.g., due to log server downtime),
+and in some cases the CA might not actually issue the corresponding certificate.
+Nevertheless, a precertificate's `signature` indicates the CA's binding intent
+to issue the corresponding certificate, which means that:
+
+* Misissuance of a precertificate is considered equivalent to misissuance of
+  the corresponding certificate. The CA should expect to be held to account,
+  even if the corresponding certificate has not actually been issued.
```

Comment:
add: "as clients have no way of verifying that the corresponding certificate has not been issued" or something along those lines?

Reply:
I think it's unnecessary to add that text.

A precertificate contains information that the CA has digitally signed.  I think it's reasonable to say that a misissued precertificate, in and of itself, is something that the CA should be held to account for (even if it can be proven that the CA has not issued the corresponding certificate).

Reply:
Acknowledged.